pilot/pkg/serviceregistry/kube/controller/network.go

```diff
@@ -306,3 +347,10 @@ func (s gatewaySet) toArray() []*model.NetworkGateway {
 	// Sort the array so that it's stable.
 	return model.SortGateways(gws)
 }
+
+func (c *Controller) isNodePortGatewayService(svc *model.Service) bool {
```

Comment:
Now we have a func and a method with the same name but different logic:

https://github.com/istio/istio/blob/ed5761ebe373cf158b16f033e69a216997ea30c9/pilot/pkg/serviceregistry/kube/controller/util.go#L191

Reply:
Removed this.